tasks/deploy/config/sepolia/morpho-blue-dai-14day.ts

```diff
@@ -0,0 +1,140 @@
+import { Address, encodeAbiParameters, parseEther, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { SEPOLIA_CHECKPOINT_REWARDER_NAME } from "./checkpoint-rewarder";
+import {
+    SEPOLIA_FACTORY_GOVERNANCE_ADDRESS,
+    SEPOLIA_FACTORY_NAME,
+} from "./factory";
+import { SEPOLIA_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const SEPOLIA_MORPHO_BLUE_DAI_14DAY_NAME = "MORPHO_BLUE_DAI_14_DAY";
+
+const CONTRIBUTION = parseEther("10000");
+
+const enc = encodeAbiParameters(
+    [
+        {
+            components: [
+                {
+                    name: "morpho",
+                    type: "address",
+                },
+                {
+                    name: "collateralToken",
+                    type: "address",
+                },
+                {
+                    name: "oracle",
+                    type: "address",
+                },
+                {
+                    name: "irm",
+                    type: "address",
+                },
+                {
+                    name: "lltv",
+                    type: "uint256",
+                },
+            ],
+            name: "MorphoBlueParams",
+            type: "tuple",
+        },
+    ],
+    [
+        {
+            morpho: "0x927A9E3C4B897eF5135e6B2C7689637fA8E2E0Bd" as `0x${string}`,
+            collateralToken:
+                "0xFF8AFe6bb92eB9D8e80c607bbe5bbb78BF1201Df" as `0x${string}`, // Hyperdrive Sepolia sDAI
+            oracle: "0x23F3A48121861b78f66BbE3DF60AD24A21c4DDad" as `0x${string}`,
+            irm: "0x0fB591F09ab2eB967c0EFB9eE0EF6642c2abe6Ab" as `0x${string}`,
+            lltv: BigInt("980000000000000000"),
+        },
+    ],
+);
```

Comment:
I'm surprised that we need to be this verbose here. Is there not a way to do this with something in the Typescript contract wrapper or artifact? It's fine if this is the only way that we can do it, I just would have expected there to be a struct type that we could encode.

Reply:
Will take a stab at resolving it from arguments like [here](https://github.com/delvtech/hyperdrive/blob/main/tasks/deploy/lib/types.ts#L210)